### ISSUE TITLE
Suppress Games button if game is not available

### ISSUE THREAD

**Issue by monzug** [alpheios-project/inflection-games#15]
in the issue #3 , I requested to suppress the Games button for conjunction etc. Actually, would it be possible to suppress the button when game is not available?

**Comment by irina060981** [comment]
Yes, I have added this ability to the last PR - i don't know what is the build you are working for - on my working server copy it looks like this for example
http://alpheios-test4.irina-sklyarova.ru/Perseus.text.1999.02.0066/book1_poem2.html

![image](https://user-images.githubusercontent.com/12377640/47223473-b9f72000-d3fc-11e8-8829-565a29a0fca8.png)

There is no Games button if there are no inflections.
There could be more situations when it has inflections but doesn't have full match and that's why there is no game there - but these more complex check - I haven't done yet.

**Comment by monzug** [comment]
I have seen it, Thanks for adding this feature. My request here is to
expand it also to word (e.g. noun, adj etc) that do not have the games
available for specific reason.
